Core/Object Arts/Dolphin/MVP/Views/Scintilla/UI.Scintilla.TextStyle.cls

```diff
@@ -29,7 +29,7 @@ Note that the `#braceHighlight` and `#mismatchedBrace` styles are normally imple
 
 With the exception of `#normal`, these styles are present to allow control over the presentation of particular features, but they cannot themselves be set against regions of text. Note that some styling, such as that used for visible whitespace, is controlled independently of the normal text style definitions  by directly setting properties of the `ScintillaView`.
 
-Up to 256 styles can be defined per view. These have symbolic names for use in Smalltalk code, and have id''s allocated automatically. The predefined styles listed above have ids which Scintilla recognises specifically, with ids between `STYLE_DEFAULT` (32) and `STYLE_LASTPREDEFINED` (39). In addition style id 0 is reserved for "normal", which (somewhat confusingly given the existance of `STYLE_DEFAULT`) is the style applied to text by default and from which all the other styles inherit any unspecified properties. See the Scintilla docs for further details of the predefined styles. All other styles are user-defined, can have any symbolic name, and are allocated an unused id in the range 1..255. 
+Up to 256 styles can be defined per view. These have symbolic names for use in Smalltalk code, and have ids allocated automatically. The predefined styles listed above have ids which Scintilla recognises specifically, with ids between `STYLE_DEFAULT` (32) and `STYLE_LASTPREDEFINED` (39). In addition style id 0 is reserved for "normal", which (somewhat confusingly given the existance of `STYLE_DEFAULT`) is the style applied to text by default and from which all the other styles inherit any unspecified properties. See the Scintilla docs for further details of the predefined styles. All other styles are user-defined, can have any symbolic name, and are allocated an unused id in the range 1..255. 
```

Comment:
id's is an abbreviation of identifiers, so should have an apostrophe to denote contraction. Or expand it.